osf/management/commands/update_mailchimp_email.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import datetime
+
+from django.core.management.base import BaseCommand
+from osf.models import OSFUser
+from website import mailchimp_utils
+
+logger = logging.getLogger(__name__)
+
+
+def update_mailchimp_email():
+    users_updated = 0
+    for user in OSFUser.objects.filter(deleted=None):
+        for list_name, subscription in user.mailchimp_mailing_lists.items():
+            if subscription:
+                mailchimp_utils.subscribe_mailchimp(list_name, user._id)
+        users_updated += 1
```

Comment:
nit: this is just going to be the number of users, right? Is there a way to be more accurate about if the address actually changed? Otherwise, this doesn't really feel worth reporting.

Reply:
If a user is unsubscribed, mailchimp will refuse the re-subscription, but the util doesn't return whether that occurred or not. Getting that info would involve modifying the util, which I don't really want to do for a run-once script.